components/HomeHero/HomeHero.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+
+import logo from '../../public/images/headless-logo.svg';
+
+import styles from './HomeHero.module.scss';
+
```

Comment:
```suggestion
import Image from 'next/image';
import logo from '../../public/images/headless-logo.svg';
import styles from './HomeHero.module.scss';
```

Slightly less readable but tidier